openupgrade_scripts/scripts/project/16.0.1.2/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Trần Trường Sơn
+# Copyright 2023 Rémy Taymans
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_translations_to_delete = [
+    "mail_template_data_project_task",
+    "project_manager_all_project_tasks_rule",
+    "project_message_user_assigned",
+    "rating_project_request_email_template",
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "project", "16.0.1.2/noupdate_changes.xml")
```

Comment:
I'd suggest not  the content of `rating_project_request_email_template` : what is the best practice on this ?

Reply:
@legalsylvain Do you have an opinion about loading new text for mail template? Should we remove it from `noupdate_changes.xml`, or should we keep it and rewrite user data?

Reply:
New texts are always added (and they usually imply changes in variables as well). And translations should be removed as well.